mscore/mixertrackitem.cpp

```diff
@@ -204,6 +204,44 @@ void MixerTrackItem::setColor(int valueRgb)
             }
       }
 
+//---------------------------------------------------------
+//   isMute
+//---------------------------------------------------------
+
+bool MixerTrackItem::isMute()
+      {
+      if (_trackType == TrackType::CHANNEL)
+            {
+            return _chan->mute();
+            }
+      const InstrumentList* il = _part->instruments();
+      bool muteFound = false;
+
+      for (auto it = il->begin(); it != il->end(); ++it)
+            {
+            Instrument* instr = it->second;
+            for (Channel *chan: instr->channel())
+                  {
+                  muteFound =  chan->mute();
+                  break;
```

Comment:
I think this is a mistake. This `break` will break the inner loop, but the instruments loop will continue.
It should probably be like this
```
for (const Instrument* instr : _part->instruments()) {
    for (const Channel *chan : instr->channel()) {
        return chan->mute();
    }
}
```

Reply:
as mentioned in an earlier review

i'm only checking if there is a instance of a channel being mute. I'm not entirely clear as how to compensate for instruments with multiple channels. From what i see the user interface doesnt have any indication of multiple channels